src/Converters.ts

```diff
@@ -1,13 +1,13 @@
 import { AzureFunctionsRpcMessages as rpc } from '../azure-functions-language-worker-protobuf/src/rpc';
 import { FunctionInfo } from './FunctionInfo';
-import { RequestBase } from './http/Request';
+import { RequestProperties } from './http/Request';
 import { Dict } from '../src/Context';
 import { BindingDefinition } from './public/Interfaces';
 
 type BindingDirection = 'in' | 'out' | 'inout';
 
-export function fromRpcHttp(rpcHttp: rpc.IRpcHttp): RequestBase {
-  let httpContext: RequestBase = {
+export function fromRpcHttp(rpcHttp: rpc.IRpcHttp): RequestProperties {
+  let httpContext: RequestProperties = {
```

Comment:
nit: let should be const

We should get you hooked up with a linting tool. That would catch things like this.